apps/remote_control/lib/lexical/remote_control/code_intelligence/entity.ex

```diff
@@ -105,6 +105,18 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Entity do
     end
   end
 
+  defp resolve({:local_arity, chars}, node_range, analysis, position) do
+    current_module = current_module(analysis, position)
+
+    case Ast.zipper_at(analysis.document, position) do
+      {:ok, %Zipper{node: {:/, _, [_, {:__block__, _, [arity]}]}}} ->
```

Comment:
same comment, should you be guarding arity as an integer between 0 and 256?

Reply:
I don't think we need this. We simply parse the correct part of the result(`capture function`) returned by `surround_context`. Based on that, we use a zipper to fetch the arity.

Reply:
ok, understood